amgut/handlers/auth_handlers.py

```diff
@@ -23,10 +23,14 @@ def get(self):
     @authenticated
     def post(self):
         skid = self.current_user
+        # log user out for register process
+        self.clear_cookie("skid")
         tl = text_locale['handlers']
-        info = {}
-        for info_column in ("email", "participantname", "address", "city",
-                            "state", "zip", "country"):
+        info = {
+            "email": self.get_argument("email"),
+            "participantname": self.get_argument("participantname")
+        }
+        for info_column in ("address", "city", "state", "zip", "country"):
             # Make sure that all fields were entered
             info[info_column] = self.get_argument(info_column, None)
```

Comment:
The comment here may be out of date. This does not make sure that all fields were entered, just makes sure that the `info` dict has a value (even if it's `None`) for address, city, state, zip, and country.